src/main/java/be/syntra/devshop/DevshopFront/services/ProductServiceImpl.java

```diff
@@ -92,7 +92,7 @@ public Product findById(Long id) {
             log.info("findById() -> product retrieved from backEnd");
             return productResponseEntity.getBody();
         } else if (HttpStatus.NOT_FOUND.equals(productResponseEntity.getStatusCode())) {
-            throw new ProductNotFoundException("Product with id = " + id + " was not found");
+            throw new ProductNotFoundException("Product with" + id + "was not found");
```

Comment:
with id: -> more readable than random number

Reply:
Product with id =5was not found -> please add the extra space between the number and was

Reply:
its only a detail but what i meant was:
if the id is 666 the error message will be '....with id =666was not...'
what we want is '....with id = 666 was not...'